manifests/config/passenger.pp

```diff
@@ -175,6 +180,67 @@
         custom_fragment         => template('foreman/_assets.conf.erb', 'foreman/_ssl_virt_host_include.erb',
                                             'foreman/_suburi.conf.erb'),
       }
+
+      $pki_dir = '/etc/pki/foreman'
```

Comment:
Good to know, searching around Debian has `/etc/ssl`. We could either set to path depending on the distro or follow the Puppet model and do something like `/usr/share/foreman/ssl` or `/var/lib/foreman/ssl` or `/etc/foreman/ssl`